tests/data_access/builders/test_todo_entry_builder.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+import pytest
+
+from src.data_access.model_builders.todo_builder import TodoBuilder
+from src.middleware.exceptions.todos import BadServerConfiguration
+
+import src.data_access.model_builders.todo_builder as todo_builder
+
+class TestTodoEntryBuilder:
```

Comment:
the convention in our team is that the name of the class is the name of the function that u want to test and the names of the functions inside are the different use cases